content/kubermatic/master/tutorials-howtos/external-clusters/aks/_index.en.md

```diff
@@ -1,24 +1,45 @@
 +++
-title = "Adding an External AKS Kubernetes Cluster"
+title = "Azure Kubernetes Service"
 date = 2022-01-10T14:07:15+02:00
-description = "Detailed tutorial to help you add an existing Kubernetes cluster in AKS and then manage it using KKP"
+description = "Detailed tutorial to help you manage AKS cluster using KKP"
 weight = 7
 
 +++
 
 ## Add AKS Cluster
 
-You can add an existing Kubernetes cluster and then manage it using KKP. From the Clusters page, click `External Clusters`.
-Click `Add External Cluster` button and Pick `Azure Kubernetes Cluster` provider.
+You can add an existing Kubernetes cluster and then manage it using KKP. 
+
+- Navigate to `External Clusters` page.
+
+- Click `Import External Cluster` button.
 
 ![Add External Cluster](/img/kubermatic/master/tutorials/external_clusters/add_external_cluster.png "Add External Cluster")
 
-Select preset with valid credentials or enter AKS `Tenant ID`, `Subscription ID`, `Client ID` and  `Client Secret`, to connect to the provider.
-The credentials should have enough access like read, write Azure Kubernetes Service and list cluster admin credential action, to fetch kubeconfig using API.
+- Pick `Azure Kubernetes Cluster` provider.
+
+![Select Provider](/img/kubermatic/master/tutorials/external_clusters/connect.png "Select Provider")
+
+- Provide Credentials in either of the below mentioned ways:
+    - Select a pre-created preset which stores the provider specific credentials. 
+      
+      Create a preset on your KKP cluster with `spec.aks.tenantID`, `spec.aks.subscriptionID`, `spec.aks.clientID` and `spec.aks.clientSecret`.
+    - Manually enter the credentials `Tenant ID`, `Subscription ID`, `Client ID` and  `Client Secret`. 
+
+{{% notice info %}}
+The credentials should have enough access like read, write Azure Kubernetes Service and list cluster admin credential, to fetch kubeconfig using API.
```

Comment:
should have enough access like read ... -> should provide access rights to read & write